### PR TITLE
[11.0] [ADD] new module mgmtsystem_action_template

### PR DESCRIPTION
**Pull request by mymage**
Ref. #263

### CODE REVIEW

**Comment by mymage** [comment]
@ivantodorovich @max3903  Can you help me to understan what is wrong with runbot? If you try, it works!

**Comment by mymage** [comment]
@ivantodorovich @max3903 Solved the runbot after the log is visible. Now it's ok and can be reviewed.

**Review by marcelofrare — APPROVED** [review]
functionally it is ok for me

**Comment by mymage** [comment]
@max3903 Hi, after a while I re-touched the PR (I learned how to do tests), but there is a problem with mgmtsystem_nonconformity_mrp. I did the PR #364 to fix that. Let me know if I can do anything else here.

File: mgmtsystem_action_template/models/mgmtsystem_action_template.py [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+    description_temp = fields.Html('Description')
+
+    @api.onchange('template_id')
+    def _onchange_template_id(self):
+        """
+        fill some fields with template ones
+        """
+
+        if self.template_id.id:
+            template = self.browse(self.template_id.id)
```
Comment (etobella) [inline mgmtsystem_action_template/models/mgmtsystem_action_template.py]: template = self.template_id would be enough

**Comment by mymage** [comment]
Code allined with 12.0 PR #351

**Review by ivantodorovich — APPROVED** [review]

**Pull request merged**